doc/user-guide/user-guide.rst

```diff
@@ -0,0 +1,39 @@
+.. _User_Guide:
+
+#######################
+Electrometer User Guide
+#######################
+
+The electrometer is a Keithley Spectrograph that measures the amount of electrons in proximity with a sensor.
+It communicates over RS-232 serial cable.
+The computer that it is connected to uses a RS-232 to USB type A cable.
+
+Electrometer Interface
+======================
+
+Link to the XML is located at the top of the :doc:`index </index>`.
+
+
+Example Use-Case
+================
+
+.. code::
+
+    from lsst.ts import salobj
+    
+    domain = salobj.Domain()
+
+    electrometer = salobj.Remote(name="Electrometer", domain=domain, index=1)
+
+    await electrometer.start_task
+
+.. code::
+
+    await electrometer.cmd_performZeroCalib.set_start(timeout=10)
+    await electrometer.cmd_setDigitalFilter.set_start(activateFilter=False, activateAvgFilter=False, activateMedFilter=False, timeout=10)
+    await electrometer.cmd_startScan.set_start(timeout=10)
+    await electrometer.cmd_stopScan.set_start(timeout=10)
+
```

Comment:
Adding instructions on how to get back the file would be helpful.

Reply:
Currently, its written to the local disk and the fits file can be found there, but I don't support the S3 LFA yet. Mainly due to the fact that it wasn't finalized at the time.

Reply:
Do you run a local server and publish the wget link?

I suggest writing a ticket to get the S3 LFA stuff done in the next few sprints.

Reply:
for this ticket just write what you do, then just create that next ticket and we're good.